docs/pages/machine-id/manifesto.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Machine ID Manifesto
+description: A manifesto for Machine Identity
+---
+
+As time goes on, machines and services are completing increasingly privileged
+work, crossing the complex boundaries between networks and organizations. AI
+agents are becoming common-place and more autonomous and powerful at an
```

Comment:
In terms of AI agents, does this mean the same thing as something like, "Organizations are entrusting AI agents in workloads that are running with less supervision but greater scope"?

Reply:
Rewritten.